app/kuma-dp/cmd/run.go

```diff
@@ -92,48 +93,27 @@ func newRunCmd() *cobra.Command {
 				runLog.Info("generated Envoy configuration will be stored in a temporary directory", "dir", tmpDir)
 			}
 
-			runLog.Info("starting Dataplane (Envoy) ...")
-
 			dataplane := envoy.New(envoy.Opts{
 				Catalog:   catalog,
 				Config:    cfg,
 				Generator: bootstrapGenerator,
 				Stdout:    cmd.OutOrStdout(),
 				Stderr:    cmd.OutOrStderr(),
 			})
+			server := accesslogs.NewAccessLogServer(cfg.Dataplane)
 
-			server := accesslogs.NewAccessLogServer()
-			defer server.Close()
-
-			logServerErr := make(chan error)
-			go func() {
-				defer close(logServerErr)
-				if err := server.Start(cfg.Dataplane); err != nil {
-					runLog.Error(err, "problem running Access Log server")
-					logServerErr <- err
-				}
-				runLog.Info("stopped Access Log server")
-			}()
-
-			dataplaneErr := make(chan error)
-			go func() {
-				defer close(dataplaneErr)
-				if err := dataplane.Run(core.SetupSignalHandler()); err != nil {
-					runLog.Error(err, "problem running Dataplane (Envoy)")
```

Comment:
And that one

Reply:
It's now handled by: `runLog.Error(err, "error while running Kuma DP")`
If any component returns an error the `componentMgr.Start` returns an error.